tests/unit/s2n_recv_test.c

```diff
@@ -197,11 +197,13 @@ int main(int argc, char **argv)
 
     /* s2n_config_set_recv_multi_record */
     {
-#define TEST_DATA_SIZE 100
-        const uint8_t test_data[TEST_DATA_SIZE] = "hello world";
-        const size_t test_data_size = sizeof(test_data);
-        uint8_t output[TEST_DATA_SIZE * 2];
-        const size_t recv_size = sizeof(output);
+        const uint8_t test_data_size = 100;
+        DEFER_CLEANUP(struct s2n_blob test_data = { 0 }, s2n_free);
+        EXPECT_SUCCESS(s2n_alloc(&test_data, test_data_size));
```

Comment:
Since the test data isn't "hello world" anymore, you might want to set it with s2n_get_public_random_data so that it's at least not just zeroes https://github.com/aws/s2n-tls/blob/main/utils/s2n_random.h#L29

Reply:
Does that really accomplish anything though? We don't check the byte values anywhere so it doesn't help for correctness checks. And before the "hello world" filled only a small fraction of the buffer so it ended up being all `0`s anyway.

Reply:
Eh the hello world meant that it at least wasn't ALL zeroes. But good point, you don't actually care what gets sent. Probably fine as-is then :)